src/ui/main.py

```diff
@@ -0,0 +1,155 @@
+import tkinter as tk
+from tkinter import ttk 
+
+class MainApplication(tk.Frame):
+    def __init__(self, parent, *args, **kwargs):
+        tk.Frame.__init__(self, parent, *args, **kwargs)
+        self.parent = parent
+
+        # duration
+        self.duration = tk.StringVar() 
+        self.duration_chosen = ttk.Combobox(root, state='readonly', textvariable = self.duration) 
+        self.duration_chosen['values'] = ('bardzo krótki',  
+                                'krótki', 
+                                'średni', 
+                                'długi')
+        self.duration_chosen.current()
+        self.duration_chosen.grid(sticky = tk.W, column = 2, row = 1, pady=10)      
+
+        self.duration_label = ttk.Label(root, text='Czas trwania wycieczki ')
+        self.duration_label.grid(sticky = tk.E, column=1, row=1)
+
+        # price
+        self.price = tk.StringVar() 
+        self.price_chosen = ttk.Combobox(root, state='readonly', textvariable = self.price)
+        self.price_chosen['values'] = ('bardzo niska',  
+                                'niska', 
+                                'umiarkowana', 
+                                'wysoka', 
+                                'bardzo wysoka')
+        self.price_chosen.current() 
+        self.price_chosen.grid(sticky = tk.W, column=2, row=2, pady=10)
+
+        self.price_label = ttk.Label(root, text='Cena ')
+        self.price_label.grid(sticky = tk.E, column=1, row=2)
+
+        # country/region
+        self.region = tk.StringVar() 
+        self.region_chosen = ttk.Combobox(root, state='readonly', textvariable = self.region)
+        self.region_chosen['values'] = ('Polska',  
```

Comment:
In the future it can be changed to directly read all unique values from database json file